packages/web/src/components/Review/Review.tsx

```diff
@@ -41,6 +42,10 @@ export const Review = (props: Props) => {
           a {props?.rating} &#x2605; review by
           <Link
             ml={1}
+            onClick={(event) => {
+              event.stopPropagation()
+              startTransition(() => navigate(`/profile/${props?.user?.username}`))
```

Comment:
without it, navigating to another route throws this:

```
A component suspended while responding to synchronous input. 
This will cause the UI to be replaced with a loading indicator. 
To fix, updates that suspend should be wrapped with startTransition.
```